.github/dependabot.yml

```diff
@@ -18,3 +18,8 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    target-branch: "v1"
```

Comment:
Is it possible to target to both `v1` and `main`?

Reply:
looks like it's not possible unfortunately 😞

Reply:
Will merging this PR to the `v1` branch work?

Reply:
I think it only works from the main branch, but let me do tests in another repo.

Reply:
@shizhMSFT Turns out you can specify the same config for multiple branches 🎉